application/views/main/phonebook/contact/add_contact.php

```diff
@@ -57,13 +56,13 @@
 <label for="is_public" style="display: inline"><?php echo lang('kalkun_public_contact_set');?></label>
 </div>
 
-<label for="group"><?php echo lang('kalkun_group'); ?></label> 
+<label for="groups"><?php echo lang('kalkun_group'); ?></label>
 <?php if (isset($contact)): ?> 
-<input name="groups" id="groups" value="<?php echo $this->Phonebook_model->get_groups($contact->row('id_pbk'), $this->session->userdata('id_user'))->GroupNames?>" type="hidden" />
+<input name="groups" id="groups" value="<?php echo $this->Phonebook_model->get_groups($contact->row('id_pbk'), $this->session->userdata('id_user'))->GroupNames?>" type="text" />
```

Comment:
Why change from hidden to text?

Reply:
the new plugin works this way. it should not be hidden

Reply:
The old plugin wasn't fit for the new jquery, so it had to be changed. So its either everything or nothing.

Reply:
Functionnally eveything is fine in the GUI, like before. What is actually the problem here?

Reply:
Hmm, let me see what I can do to handle this scenario.

Reply:
BTW, the motivation of the changes are given in each individual commit (that's why I ask to not squash & merge the PRs). When you review by commit you see the comment (which is not the case when you review by file)